src/Game.js

```diff
@@ -1,32 +1,41 @@
 import Player from "./Player";
 import Round from "./Round";
+import data from "./data";
+import Puzzle from "./Puzzle";
 
 class Game {
   constructor() {
     this.players = [];
     this.round = 0;
     this.puzzleSet = [];
   }
+
   createPlayers(name1, name2, name3) {
-    // take inputs of p1, p2, and p3
-    // pushing inputs into players array
     let p1 = new Player(name1, 1);
     let p2 = new Player(name2, 2);
     let p3 = new Player(name3, 3);
     this.players.push(p1, p2, p3);
   }
 
-  getPuzzleSet() {
-    // get into data filter, obj puzzles, each obj of puzzles, key of puzzle_bank
-    // randomly grab 5
-    // push 5 into this.puzzleSet
+  getPuzzles() {
+    let allPuzzles = [];
+    Object.keys(data.puzzles).forEach(type => {
+      data.puzzles[type].puzzle_bank.forEach(puzzle => {
+        allPuzzles.push(puzzle);
+      });
+    });
+    shufflePuzzles(allPuzzles);
+  }
 
-    // data drill maybe data.puzzles[numAnswers].filter
+  shufflePuzzles(puzzles) {
+    puzzles.sort() => .5 - Math.random();
+    this.puzzleSet = puzzles.slice(0, 4);
   }
   
   createRound() {
     let wheel = new Wheel();
```

Comment:
This all looks really good. We just need to make sure that before we pass in the instance of wheel, that we run the method that gets the values and shuffles it.